Neos.Eel/Classes/Helper/StringHelper.php

```diff
@@ -567,7 +567,7 @@ public function md5($string)
      */
     public function length($string)
     {
-        return UnicodeFunctions::strlen($string);
+        return $string === null ? 0 : UnicodeFunctions::strlen($string);
```

Comment:
Since UnicodeFunctions no longer allow calling the methods with null subject (which btw. never was possible on HHVM - see https://3v4l.org/DUFhF#v560), but there are Eel StringHelper tests that expect length to allow null, this check is put in place. Anyone knows if Eel StringHelper has more such assumptions that the subject *may be* `null`?